### DISCUSSION TITLE
Cypress was killed with SIGTERM (Termination)

### DISCUSSION THREAD

**Discussion by davidjeyachandran** [cypress-io/cypress#22440] [Questions and Help]
Cypress 10 has been working beautifully and it stopped working yesterday afternoon. It's not a bug with Cypress because I've tested the same repository on other machines. I think Cypress stopped abruptly and now I'm getting the following error: 

```
Cypress failed to start.
This may be due to a missing library or dependency. https://on.cypress.io/required-dependencies
Please refer to the error below for more details.
----------
Command was killed with SIGTERM (Termination): /Users/david.jeyachandran/Library/Caches/Cypress/10.2.0/Cypress.app/Contents/MacOS/Cypress --no-sandbox --smoke-test --ping=393
----------
Platform: darwin-arm64 (21.4.0)
Cypress Version: 10.2.0
```

I've tried:

- rm -rf node_modules
- rm package-lock.json
- rm -rf ~/Library/Caches/Cypress
- restarting my machine
- npm install
- npx cypress open // Error appears

I've also run cypress with the debug flag & I'm happy to post this but it didn't provide me any clues.
DEBUG=cypress:cli npx cypress open

I then tried installing different Cypress versions (10.2.0, 10.1.0, 10.03). I know it's not the version because 10.1 and 10.2 were working fine on my machine. I tried the same repository on another machine and just did an `npm install` and `npx cypress open` and it just worked.

Any ideas? I think I've completely uninstalled Cypress and reinstalled it. Could SIGTERM keep a track of processes that have failed and immediately kills Cypress.

**Comment by hamletfg** [comment]
What fixed it for me was that I installed Cypress 10.2.0 in a subdirectory, that wouldn't work for some reason, but when I installed Cypress in my username director it was able to run in my coding subdirectory. Let me know if that works!

Edit: the install was done in terminal

**Reply by davidjeyachandran** [reply]
Thank you for responding. How did you install Cypress in your username directory (home directory)? I tried using `npm install cypress -g` but I hit the same error.

**Comment by hamletfg** [comment]
I used ```npm install cypress --save-dev``` and the guide on [Cypress' website ](https://docs.cypress.io/guides/getting-started/installing-cypress#What-you-ll-learn)
